website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -7,36 +7,34 @@ description: >
 ---
 
 Karpenter sets a Kubernetes [finalizer](https://kubernetes.io/docs/concepts/overview/working-with-objects/finalizers/) on each node it provisions.
-The finalizer specifies additional actions the Karpenter controller will take in response to a node deletion request.
-These include:
-
-* Marking the node as unschedulable, so no further pods can be scheduled there.
-* Evicting all pods other than daemonsets from the node.
-* Terminating the instance from the cloud provider.
-* Deleting the node from the Kubernetes cluster.
+The finalizer signals to the Karpenter Termination controller to gracefully shutdown a node that receives a deletion request. Karpenter deprovisioning can be executed by Karpenter through the Deprovisioning Controler or by the user through manual deprovisioning.
+
+### Deprovisioning Controller
+Karpenter automatically discovers deprovisionable nodes and spins up replacements when needed. Karpenter deprovisions nodes by executing one [automatic method](#methods) at a time, in order of Expiration, Drift, Emptiness, then Consolidation. Each method varies slightly but they all follow the standard deprovisioning process:
```

Comment:
The methods are called out below and then expanded on even further. I don't include interruption here because it only follows the termination controller, not the deprovisioning controller.